vendor/assets/stylesheets/susy/susy/language/susy/_settings.scss

```diff
@@ -0,0 +1,216 @@
+// Susy Settings
+// =============
+
+// Susy Language Defaults
+// ----------------------
+// - PRIVATE
+@include susy-defaults((
+  container: auto,
+  math: fluid,
+  output: float,
+  container-position: center,
+  gutter-position: after,
+  global-box-sizing: content-box,
+  debug: (
+    image: hide,
+    color: rgba(#66f, .25),
```

Comment:
Color literals like `#66f` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.25` should be written with a leading zero as `0.25`